_ml-commons-plugin/index.md

```diff
@@ -10,7 +10,7 @@ has_toc: false
 
 ML Commons for OpenSearch eases the development of machine learning features by providing a set of common machine learning (ML) algorithms through transport and REST API calls. Those calls choose the right nodes and resources for each ML request and monitors ML tasks to ensure uptime. This allows you to leverage existing open-source ML algorithms and reduce the effort required to develop new ML features.
 
-Interaction with the ML Commons plugin occurs through either the [REST API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api) or [AD]({{site.url}}{{site.baseurl}}/observability-plugin/ppl/commands#ad) and [kmeans]({{site.url}}{{site.baseurl}}/observability-plugin/ppl/commands#kmeans) Piped Processing Language (PPL) commands.
+Interaction with the ML Commons plugin occurs through either the [REST API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api) or [AD]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/functions#ad) and [kmeans]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/functions#kmeans) Piped Processing Language (PPL) commands.
```

Comment:
i think i brought this up previously but couldn't remember the conclusion. SQL is under `search` category and PPL is under `observability` category because they have different use cases from user's perspective. is there a reason to move PPL out of `observability`?

Reply:
I agree with you -- the use cases are different. I moved to SQL, because it is implemented by SQL plugin. I'm welcome for proposals and ready to move it back.

Reply:
got it. I don't have a strong opinion on this, i think it's ok if no objections